extras/indexing/src/main/java/org/apache/rya/indexing/accumulo/ConfigUtils.java

```diff
@@ -453,9 +458,24 @@ public static void setIndexers(final RdfCloudTripleStoreConfiguration conf) {
             optimizers.add(StatementMetadataOptimizer.class.getName());
         }
 
-        conf.setStrings(AccumuloRdfConfiguration.CONF_ADDITIONAL_INDEXERS, indexList.toArray(new String[] {}));
-        conf.setStrings(RdfCloudTripleStoreConfiguration.CONF_OPTIMIZERS, optimizers.toArray(new String[] {}));
+        final String[] existingIndexers = conf.getStrings(AccumuloRdfConfiguration.CONF_ADDITIONAL_INDEXERS);
```

Comment:
this is another issue, where if you set the indexers on the config before initializing them, the indexers get overwritten